doc/espnet2_tutorial.md

```diff
@@ -0,0 +1,176 @@
+    # Doesn't work
+    cd egs2/an4/asr1/local/
+    ./data.sh
+    
+    # Work
+    cd egs2/an4/asr1
+    ./run.sh
+    ./scripts/<some-script>.sh
+    ```
+    
+1. Change the configuration
+    Describing the directory structure as follows:
+    
+    ```
+    egs2/an4/asr1/
+     - conf/      # Configuration files for training, inference, etc.
+     - scripts/   # Bash utilities of espnet2
+     - pyscripts/ # Python utilities of espnet2
+     - steps/     # From Kaldi utilities
+     - utils/     # From Kaldi utilities
+     - db.sh      # The directory path of each corpora
+     - path.sh    # Setup script for environment variables
+     - cmd.sh     # Configuration for your backend of job scheduler
+     - run.sh     # Entry point
+     - asr.sh     # Invoked by run.sh
+    ```
+
+    - You need to modify `db.sh` for specifying your corpus before executing `run.sh`. For example, when you touch the recipe of `egs2/wsj`, you need to change the paths of `WSJ0` and `WSJ1` in `db.sh`.
+      - Some corpora can be freely obtained from the WEB and they are written as "downloads/" at the initial state. You can also change them to your corpus path if it's already downloaded.
+    - `path.sh` is used to set up the environment for `run.sh`. Note that the Python interpreter used for ESPnet is not the current Python of your terminal, but it's the Python which was installed at `tools/`. Thus you need to source `path.sh` to use this Python.
+        ```bash
+        source path.sh
+        python
+        ```
+    - `cmd.sh` is used for specifying the backend of the job scheduler. If you don't have such a system in your machine environment, you don't need to change anything about this file. See [Using Job scheduling system](./parallelization.md)
+
+1. Run `run.sh`
+
+    ```bash
+    ./run.sh
+    ```
+
+    `run.sh` is an example script, which we often call as "recipe", to run all stages related to DNN experiments; data-preparation, training, and evaluation.
+
+## See training status
+
+### Show the log file
+
+```bash
+% tail -f exp/asr_train_<some-name>/train.log
+[host] 2020-04-05 16:34:54,278 (trainer:192) INFO: 2/40epoch started. Estimated time to finish: 7 minutes and 58.63 seconds
+[host] 2020-04-05 16:34:56,315 (trainer:453) INFO: 2epoch:train:1-10batch: iter_time=0.006, forward_time=0.076, loss=50.873, los
+s_att=35.801, loss_ctc=65.945, acc=0.471, backward_time=0.072, optim_step_time=0.006, lr_0=1.000, train_time=0.203
+[host] 2020-04-05 16:34:58,046 (trainer:453) INFO: 2epoch:train:11-20batch: iter_time=4.280e-05, forward_time=0.068, loss=44.369
+, loss_att=28.776, loss_ctc=59.962, acc=0.506, backward_time=0.055, optim_step_time=0.006, lr_0=1.000, train_time=0.173
+```
+
+### Show the training status in a image file
+
+```bash
+# Accuracy plot
+# (eog is Eye of GNOME Image Viewer)
+eog exp/asr_train_<some-name>/images/acc.img
+# Attention plot
+eog exp/asr_train_<some-name>/att_ws/<sample-id>/<param-name>.img
+```
+
+### Use tensorboard
```

Comment:
Maybe better to add some descriptions about how to install tensorboard and a simple usage like https://espnet.github.io/espnet/tutorial.html#logging.
We don't include the tensorboard installation as default (we could do it though).

Reply:
Maybe it is OK to leave it as it is but in the near future, we may need to prepare a complete README for espnet2 that merges both espnet1 and espnet2 documentations.

Reply:
Sorry, I already added tensorboard.

https://github.com/espnet/espnet/blob/eca4fa073a826957c91069501844d4e2b9392cc9/setup.py#L33-L34

This is  because pytorch1.1? officially start to support tensorboard logging and it requires to install tensorboard. 
I decided to use the pytorch's SummaryWriter instead of tensorboardx if pytorch>=1.1, so we need to install tensorboard.

Reply:
Make sense for me.
This is very good.